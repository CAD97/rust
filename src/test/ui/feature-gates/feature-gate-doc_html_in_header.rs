#![doc(html_in_header = "head.html")] //~ ERROR: `#[doc(html_in_header)]` is experimental
fn main() {}
