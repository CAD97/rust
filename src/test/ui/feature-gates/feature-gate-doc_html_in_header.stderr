error[E0658]: `#[doc(html_in_header)]` is experimental
  --> $DIR/feature-gate-doc_html_in_header.rs:1:1
   |
LL | #![doc(html_in_header = "head.html")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: add `#![feature(doc_html_in_header)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
